event_registration_cancel_reason/views/event_registration_view.xml

```diff
@@ -22,8 +22,8 @@
         <field name="model">event.registration</field>
         <field name="inherit_id" ref="event.view_registration_search"/>
         <field name="arch" type="xml">
-            <filter context="{'group_by':'state'}" position="after">
-                <filter string="Cancellation reason" domain="[]" context="{'group_by': 'cancel_reason_id'}"/>
+            <filter name="status" context="{'group_by':'state'}" position="after">
```

Comment:
Then remove `context` attribute here in the selector.

Reply:
Done @pedrobaeza